<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ML Substation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line_horizontal_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>791</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_horizontal_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>791</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_case">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Case Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_case">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>611</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Please choose a case.</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Status&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_description">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Description&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_des">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>771</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_create">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>97</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_edit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>470</y>
      <width>97</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>View/Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dupilicate">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>470</y>
      <width>97</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Duplicate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_request">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>97</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Request</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_subscribe">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>510</y>
      <width>97</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Subscribe</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_show">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>510</y>
      <width>97</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Results</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Status">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>60</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;N/A&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>814</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox_case</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>caseChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_create</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>create()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_edit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>edit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_show</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>result()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>551</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>caseChanged()</slot>
  <slot>create()</slot>
  <slot>edit()</slot>
  <slot>result()</slot>
 </slots>
</ui>
